common/djangoapps/util/enterprise_api.py

```diff
@@ -0,0 +1,62 @@
+"""
+Utilities to communicate with the Enterprise service.
+"""
+import logging
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from edx_rest_api_client.client import EdxRestApiClient
+from slumber.exceptions import HttpClientError, HttpServerError
+
+from openedx.core.lib.token_utils import JwtBuilder
+
+
+log = logging.getLogger(__name__)
+
+
+class EnterpriseApiException(Exception):
+    """
+    Exception for errors while communicating with the Enterprise service API.
+    """
+    pass
+
+
+class EnterpriseApiClient(object):
+    """
+    Class for producing an Enterprise service API client.
+    """
+
+    def __init__(self):
+        """
+        Initialize an Enterprise service API client, authenticated using the Enterprise worker username.
+        """
+        self.user = User.objects.get(username=settings.ENTERPRISE_SERVICE_WORKER_USERNAME)
+        jwt = JwtBuilder(self.user).build_token([])
+        self.client = EdxRestApiClient(
+            settings.ENTERPRISE_API_URL,
```

Comment:
for settings references we should consider piping through `openedx.core.djangoapps.site_configuration.helpers` -- lots of examples of this in the code.